bytes/bytes/raw/file_raw_repository.py

```diff
@@ -62,3 +72,68 @@ def _raw_file_path(self, raw_id: UUID, boefje_meta: BoefjeMeta) -> Path:
 
     def _index(self, raw_id: UUID) -> str:
         return str(raw_id)[: self.UUID_INDEX]
+
+
+class S3RawRepository(RawRepository):
+    def __init__(
+        self,
+        file_middleware: FileMiddleware,
+        bucket_per_org: bool,
+        s3_bucket_prefix: str | None = "OpenKAT-",
+        s3_bucket_name: str | None = "OpenKAT",
+    ) -> None:
+        self.file_middleware = file_middleware
+        self.bucket_per_org = bucket_per_org
+        self.s3_bucket_prefix = s3_bucket_prefix
+        self.s3_bucket_name = s3_bucket_name
+
+        self._session: Session = None
+        self._s3resource = None
+
+    @property
```

Comment:
I suggest using `@functools.cached_property`, this will cache the created S3 resource. Or leave it as is

Reply:
Is that needed? Since the method "caches" the property (`_s3resource`) already

Reply:
Or do you think it is better to let `@functools.cached_property` handle the caching?

Reply:
It's not really better in the sense of a better cache utility, but it's simpler. Although it's a common pattern, it wasn't immediately clear that the property is cached; whereas using the decorator would make it more obvious. But it's a gray area, so I'll leave it up to you. Both approaches are valid

Reply:
since the attribute `_s3resource` never gets used outside of the property-method. I think it is indeed better to use the property-method exclusively. And make it a `@functools.cached_property`. And the original  `_s3resource` will be removed